test-cases/performance/perf-regression-gcp.100threads.30M-keys.yaml

```diff
@@ -0,0 +1,39 @@
+test_duration: 300
+
+stress_cmd_w: "cassandra-stress write no-warmup cl=QUORUM duration=60m -schema 'replication(factor=3)' -mode cql3 native -rate threads=100 -pop seq=1..30000000"
+prepare_write_cmd: "cassandra-stress write no-warmup cl=QUORUM n=30000000 -schema 'replication(factor=3)' -mode cql3 native -rate threads=100 -pop seq=1..30000000"
+stress_cmd_r: "cassandra-stress read no-warmup cl=QUORUM duration=50m -schema 'replication(factor=3)' -mode cql3 native -rate threads=100 -pop 'dist=gauss(1..30000000,15000000,1500000)' "
+stress_cmd_m: "cassandra-stress mixed no-warmup cl=QUORUM duration=50m -schema 'replication(factor=3)' -mode cql3 native -rate threads=100 -pop 'dist=gauss(1..30000000,15000000,1500000)' "
+stress_multiplier: 2
+
+# NOTE: following is needed for the K8S case
+k8s_loader_run_type: 'static'
+
+n_db_nodes: 3
+n_loaders: 4
+n_monitor_nodes: 1
+
+
+use_preinstalled_scylla: true
+gce_instance_type_db: 'n2-highmem-8'
+gce_root_disk_type_db: 'pd-ssd'
+gce_n_local_ssd_disk_db: 8
+
+gce_instance_type_loader: 'c3-highcpu-8'
+gce_root_disk_type_loader: 'pd-ssd'
+root_disk_size_loader: 50
+
+gce_instance_type_monitor: 'n2-highmem-8'
```

Comment:
IIUC all the above is the same as the original test yaml.
If so, why not just creating a configuration for gce and use it as another configuration file for the pipeline?

Reply:
maybe we should not include perf-search-best-throughput-configs at ass as it will be used only once and at next run should be changed anyway. @roydahan What do you think

Reply:
after removing duplicates [perf-search-best-throughput-config-gcp.yaml](https://github.com/scylladb/scylla-cluster-tests/pull/6294/files#diff-22a3d171e726ad0742c12fc54f7219ed410f8a39de351bdbbe19d3290be0ac41) config file will looks like:
```test_duration: 3000

stress_cmd_w: "cassandra-stress write no-warmup cl=QUORUM duration=30m -schema 'replication(factor=3)' -mode cql3 native -rate threads=50 -pop seq=1..30000000"
prepare_write_cmd: "cassandra-stress write no-warmup cl=QUORUM n=30000000 -schema 'replication(factor=3)' -mode cql3 native -rate threads=100 -pop seq=1..30000000"
stress_cmd_r: "cassandra-stress read no-warmup cl=QUORUM duration=30m -schema 'replication(factor=3)' -mode cql3 native -rate threads=50 -pop 'dist=gauss(1..30000000,15000000,1500000)' "
stress_cmd_m: "cassandra-stress mixed no-warmup cl=QUORUM duration=30m -schema 'replication(factor=3)' -mode cql3 native -rate threads=50 -pop 'dist=gauss(1..30000000,15000000,1500000)' "
stress_multiplier: 1

n_loaders: 8


num_loaders_step: 1
stress_threads_start_num: 50
num_threads_step: 25
stress_step_duration: '15m'
max_deviation: 2 # 5% relative difference
n_stress_process: 1


user_prefix: 'perf-search'
print_kernel_callstack: false
store_perf_results: false
enable_argus: false
```

for me, it is not readable and I would prefer to save duplication for easy-read configuration from 1 file instead of looking into multiple config files and searching which params were added and which override 

@roydahan if you still prefer to use 2 confides I will do it.

Reply:
Why config will look like this?
I thought it should be identical to the original job.

I agree it doesn't make sense to have too many configurations overriding, but I don't understand why they're not identical.
The thumb rule should be:
If identical, 1 yaml + config for configuration.
else, each has its own yaml.

Reply:
> Why config will look like this? I thought it should be identical to the original job.
> 
> I agree it doesn't make sense to have too many configurations overriding, but I don't understand why they're not identical. The thumb rule should be: If identical, 1 yaml + config for configuration. else, each has its own yaml.

because it is a different test

in perf-search-best-throughput we run many small tests (so stress commands are different) with different numbers of loaders/processes/treads, with disabled reporting, Argus, elastic etc.